pkg/apis/crd/v1alpha1/types.go

```diff
@@ -354,10 +354,17 @@ type Rule struct {
 	// +optional
 	From []NetworkPolicyPeer `json:"from"`
 	// Rule is matched if traffic is intended for workloads selected by
-	// this field. If this field is empty or missing, this rule matches all
-	// destinations.
+	// this field. This field can't be used with ToServices. If this field
+	// and ToServices are both empty or missing this rule matches all destinations.
 	// +optional
 	To []NetworkPolicyPeer `json:"to"`
+	// Rule is matched if traffic is intended for workloads selected by
+	// this field. Currently only ClusterIP types Services are supported
```

Comment:
This is ambiguous IMO. We should use `Rule is matched if traffic is intended for a Service listed in this field`. The current phrasing *kind of* implies that we would enforce the policy when accessing endpoints directly, which is not the case.

Reply:
Done.